spec/support/i18n.rb

```diff
@@ -1,20 +1,40 @@
 RSpec.configure do |config|
-  # Check that the factories are all valid.
   config.before(:suite) do
-    # The Stubbed I18n backend will return the translation key, while having the same behaviour
-    # as the standard backend. This allows tests to be written without assuming the translation.
+    # The Stubbed I18n backend will allow certain translation keys to be returned directly, ignoring
+    # the presence (or absence) of actual translations. This allows tests to be written without
+    # assuming the translation.
     class StubbedI18nBackend < I18n::Backend::Simple
       protected
 
       def lookup(_, key, _, _)
         key = key.to_s
         result = super
-        if key.start_with?('activerecord', 'attributes', 'errors', 'support')
+        if always_return_key?(key)
+          key
+        elsif always_return_actual?(key) || result.nil?
```

Comment:
Some keys will be translations, because Rails would use them. Others we will stub them.